.evergreen/utils.sh

```diff
@@ -28,12 +28,8 @@ createvirtualenv () {
         . $VENVPATH/bin/activate
     fi
 
-    PYVER=$(${PYTHON} -c "import sys; sys.stdout.write('.'.join(str(val) for val in sys.version_info[:2]))")
-    # pip fails to upgrade in a Python 3.6 venv on Windows.
```

Comment:
This failure no longer happens so when can always upgrade pip now. Upgrading pip resolves the test failures.